src/libs/ajax/surveys/Surveys.ts

```diff
@@ -0,0 +1,8 @@
+import * as qs from 'qs';
+import { fetchGoogleForms } from 'src/libs/ajax/ajax-common';
+
+export const Surveys = (signal?: AbortSignal) => ({
+  submitForm: (formId, data) => fetchGoogleForms(`${formId}/formResponse?${qs.stringify(data)}`, { signal }),
```

Comment:
The comment on the PR indicates that you intended to remove this.

Reply:
oops!  I updated PR description... that was a left-over from the last PR